.github/workflows/linux.yml

```diff
@@ -2,7 +2,9 @@ name: Linux
 
 on:
   pull_request:
+    paths: "!**.yml"
```

Comment:
This doesn't seem like the best rule, since it would prevent github actions from executing when someone only changed linux.yml, for example.